### DISCUSSION TITLE
Why are plex_search and collection_order incompatible?

### DISCUSSION THREAD

**Discussion by benfishbus** [meisnate12/Plex-Meta-Manager#477] [Q&A] [answered]
Is there any way to make them work together, even if ugly? Situation: I want each library's "Christmas" collection to be able to display the contents of the others', **_and_** I want to sort by critic rating. The former only works with static collections, whereas the latter only works with smart collections. Any way to resolve?

**Comment by meisnate12** [comment, marked as answer]
it just an oversite that custom sort doesn't work with plex_search try the latest develop it should work now

**Reply by benfishbus** [reply]
Working great! Thank you.
